clnt/client.go

```diff
@@ -0,0 +1,260 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net"
+	"os"
+	"time"
+
+	pb "simple-worker/protobuf"
+
+	"github.com/google/uuid"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var conn *grpc.ClientConn
```

Comment:
changed it in latest commit. For a simple client program, it would have been ok to have a global variable though.

I added client information to the command structures (for each command). This is one way of doing it.
Another way would be to call connect() in run() of each command but if connection has issues, it wouldn't report it immediately.